### PR TITLE
qt5.8: fix crash with multidisplay environment

### PR DESCRIPTION
**Pull request by cgrzemba**
These two patches fix crash problem if QT application is started on other display than :0
Fix for openindiana issue 12301

### CODE REVIEW

**Comment by alarcher** [comment]
@cgrzemba Are you able to add a link to the corresponding ticket in each patch? I'll merge right after.

**Comment by cgrzemba** [comment]
Do you mean the QT tickets?

**Comment by alarcher** [comment]
> Do you mean the QT tickets?

Yes if it is possible, just to be able to track the issues when we update Qt. Thank you.

File: components/library/qt5/patches/qtbase-0005-qglxconvience-qtbug81904.patch [hunk 1]
```diff
@@ -1,67 +1,35 @@
...
++        if (pointer)
++            XFree(pointer);
+     }
+ };
+ 
+@@ -202,6 +203,8 @@ GLXFBConfig qglx_findConfig(Display *display, int screen , QSurfaceFormat format
              GLXFBConfig candidate = configs[i];
  
              QXlibPointer<XVisualInfo> visual(glXGetVisualFromFBConfig(display, candidate));
--
```
Comment (alarcher) [inline components/library/qt5/patches/qtbase-0005-qglxconvience-qtbug81904.patch]: Did you drop this part of the patch on purpose? Just to confirm.

Comment (cgrzemba) [reply components/library/qt5/patches/qtbase-0005-qglxconvience-qtbug81904.patch]: yes, it is now the original patch frm QT 5.9

**Pull request merged**